### ISSUE TITLE
Request for comments: "Secure" releases - Any idea on how to resolve False positives on VirusTotal

### ISSUE THREAD

**Issue by sylikc** [sylikc/jpegview#113]
So, I was perusing Reddit links about JPEGView and I noticed that there are a few mentions that 1.0.40 reported to be a virus in 7z form, but when someone re-zip'd it up, it was fine.  Anyone have experience with this?

I'm thinking that now that there are regular "nightlies" built, and that the interest is picking up again, people would inevitably copy the source and possibly make modified installers, some of which are good, others are ... well, not so good.

* The easiest way would obviously be code signing.  But it costs $$$.
* Would PGP signing releases be an alternative?
* How about checksums of released binary files (inside the directory) which are then PGP signed?

What's the "industry" standard way that's widely accepted?  I have seen lots of `.sig` files for releases of other projects... but I am still debating the best approach so people know what releases they're getting

**Comment by pureby** [comment]
Yes, false-positives for non-signed software can sadly be an obstacle for free software development sometimes.

The easiest method to ensure your users are getting the official, unmodified packages is probably to simply publish official version of your software to as many repo's as possible (Winget, Chocolatey, Scoop, PortableApps), along with publishing SHA-256 checksums for installer/exe/zip in the release notes on GitHub. Then simply include (a reference to) those checksums in README, on your website, and in package description in every repo you publish to.

Of course, this way you leave it up to the user to download the software from official source and to do the referencing. But at least you made your best effort to make your software is officially accessible through popular repositories, and you provide users with a bulletproof way to easily verify file integrity if so needed. Plus, it does not cost a penny.

**Comment by pureby** [comment]
Another idea just came to mind: since you have already improved the application significantly, and from what I can read from your posts you are re-thinking a lot of the defaults, settings, and there is even a totally new visual enhancement coming up, why not release the next major update (any time in the future) under a new name, something like _JPEGView 2_ or _JPEGView Silk_? 

This will give you the benefit of discoverability from people searching for JPEGView and at the same time distinguish your product from the abandoned version.

(Please excuse me for the wordplay on your username, it's just something that came to mind when thinking of an easy app name).
